client/client-multi/client-objective-c/Kaa/avro/gen/EndpointGen.h

```diff
@@ -24,6 +24,7 @@
 #import "KAAUnion.h"
 
 
+
```

Comment:
This file is auto-generated, but as I understood I put a new line manually, am I correct? Do we need it?